frontend/src/app/components/project-list/project-list.component.ts

```diff
@@ -1,105 +1,52 @@
 import { Component, OnInit } from '@angular/core';
 import { faMagnifyingGlass } from '@fortawesome/free-solid-svg-icons';
+import { ProjectInfoModel } from 'src/core/models/project/project-info-model';
+import { ProjectModel } from 'src/core/models/project/project-model';
 import { UserModel } from 'src/core/models/user/user-model';
-import { ProjectModel } from '../../../core/models/project/project-model';
+import { GetCurrentOrganizationService } from 'src/core/services/get-current-organization.service';
+import { ProjectService } from 'src/core/services/project.service';
 
 @Component({
   selector: 'app-project-list',
   templateUrl: './project-list.component.html',
   styleUrls: ['./project-list.component.sass']
 })
 export class ProjectListComponent implements OnInit {
-  public currentUser: UserModel = {
-    id: 0,
-    name: '',
-    email: ''
-  };
+  
+  public currentUserModel: UserModel;
 
   public inputSearch = '';
   public searchIcon = faMagnifyingGlass;
 
-  public items: ProjectModel[] = [
-    {
-      id: 1,
-      name: 'Tasque',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-    {
-      id: 2,
-      name: 'Youtube',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-    {
-      id: 3,
-      name: 'Facebook',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-    {
-      id: 4,
-      name: 'Twitch',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-    {
-      id: 5,
-      name: 'AWS',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-    {
-      id: 6,
-      name: 'Dia',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-    {
-      id: 7,
-      name: 'Spotify',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-    {
-      id: 8,
-      name: 'Google Chrome',
-      authorId: 1,
-      organizationId: 1,
-      createdAt: new Date(Date.now()),
-      updatedAt: new Date(Date.now()),
-    },
-  ];
+  public projectsModel: ProjectInfoModel[] = [];
+  public items: ProjectModel[] = [];
 
-  public itemsShow = this.items;
+  public itemsShow = this.projectsModel;
 
-  constructor() { }
+  constructor(public projectService: ProjectService, 
+    private currentOrganization: GetCurrentOrganizationService) { 
+      
+  }
 
   ngOnInit(): void {
+    this.projectService.getAllProjectsOfThisOrganization(this.currentOrganization.currentOrganizationId).subscribe((data) => {
```

Comment:
Also not sure what projects should load: 
- Only users projects in the current organization?
- Or all projects?
I think, we should decide it on the tomorrow meeting.

Reply:
We was discussed this, all projects of the organization to which the current user has access. In the task of refactoring after adding roles, it will be good to limit this